samba/smb.conf

```diff
@@ -13,6 +15,47 @@
    interfaces = %%INTERFACE%%
    hosts allow = %%ALLOW_HOSTS%%
 
-   #guest account = root
-   #map to guest = Bad Password
-   #ntlm auth = yes
+[config]
+   browseable = yes
+   writeable = yes
+   path = /config
+
+   valid users = %%USERNAME%%
+   force user = root
+   force group = root
+
+[addons]
+   browseable = yes
+   writeable = yes
+   path = /addons
+
+   valid users = %%USERNAME%%
+   force user = root
+   force group = root
+
+[ssl]
+   browseable = yes
+   writeable = yes
+   path = /ssl
+
+   valid users = %%USERNAME%%
+   force user = root
+   force group = root
+
+[share]
+   browseable = yes
+   writeable = yes
+   path = /share
+
+   valid users = %%USERNAME%%
+   force user = root
+   force group = root
+
+[backup]
+   browseable = yes
+   writeable = yes
+   path = /share
```

Comment:
@pvizeli It looks like the `[share]` and `[backup]` shares have the same path.
Should this be `/backup` instead?

Reply:
I think you are correct, opened PR https://github.com/home-assistant/hassio-addons/pull/439